docs/src/examples/components/Popup/Usage/PopupExampleOnWithFocusTrap.shorthand.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
+import { Button, Popup, Grid, Text, popupFocusTrapBehavior } from '@stardust-ui/react'
+
+const contentWithButtons = {
+  content: (
+    <>
+      <Button onClick={() => alert('1')}>First</Button>
```

Comment:
Looks like a forgotten debug statement

Reply:
We want to show here that opening alert will not close the popup, that's why we have this, but I am okay with removing it, or changing the content of it. What do you think?

Reply:
My opinion: we should avoid the usage of `alert()` in docs.

Reply:
Removing it, it didn't bring much value anyways :)